pkg/agent/route/route_linux.go

```diff
@@ -115,19 +116,46 @@ func (c *Client) Initialize(nodeConfig *config.NodeConfig, done func()) error {
 func (c *Client) initIPTablesOnce(done func()) {
 	defer done()
 	backoffTime := 2 * time.Second
+	stopCh := signals.RegisterSignalHandlers()
+	c.syncIPTablesWithRetry(backoffTime)
+	klog.Info("Successfully synced iptables")
+	klog.Info("Starting sync loop for IPTables")
+	go c.ipTablesSyncLoop(stopCh)
+	return // FIXME: redundant
+}
+
+func (c *Client) ipTablesSyncLoop(stopCh <-chan struct{}) {
+	defer klog.Infof("Stopping IPTables SyncLoop")
+	var syncPeriod = 30 * time.Second
+	var syncTicker = time.NewTicker(syncPeriod)
+	defer syncTicker.Stop()
+	for {
+		select {
+		case <-stopCh:
+			return
+		case <-syncTicker.C:
+			if err := c.syncIPSet(); err != nil {
+				klog.Errorf("Failed to sync ipset: %v - will retry in %v", err, syncPeriod)
+				continue
+			}
+			c.syncIPTablesWithRetry(5 * time.Second)
```

Comment:
Pleas make the sync period a top-level named `const`

Reply:
Moved this to top level constant in agent.go, so that we can control the sync interval. For ex- integration tests uses a much smalled interval to speed up it's sync test. Currently, I have kept sync interval at 60 sec. Let me know what value do you suggest?